apps/test/integration/levelSolutions/applab2/ec_simple.js

```diff
@@ -139,85 +112,6 @@ module.exports = {
       expect: '"3 4 b,c,d,f"'
     }),
 
-    // These exercise all of the blocks in canvas category
-    // It does not validate that they behave correctly, just that we don't end
-    // up with any errors
-    testApplabConsoleOutput({
-      testName: 'Canvas',
-      source: `
-        createCanvas("id", 320, 480);
-        setActiveCanvas("id");
-        line(0, 0, 160, 240);
-        circle(160, 240, 100);
-        rect(80, 120, 160, 240);
-        setStrokeWidth(3);
-        setStrokeColor("red");
-        setFillColor(rgb(255,0,0));
-        drawImage("id", 0, 0);
-        var imgData = getImageData(0, 0, 320, 480);
-        putImageData(imgData, 0, 0);
-        clearCanvas();
-        getRed(imgData, 0, 0);
-        getGreen(imgData, 0, 0);
-        getBlue(imgData, 0, 0);
-        getAlpha(imgData, 0, 0);
-        setRed(imgData, 0, 0, 255);
-        setGreen(imgData, 0, 0, 255);
-        setBlue(imgData, 0, 0, 255);
-        setAlpha(imgData, 0, 0, 255);
-        setRGB(imgData, 0, 0, 255, 255, 255);
-      `,
-      expect: ''
-    }),
-
-    // These exercise all of the blocks in Turtle category
-    // It does not validate that they behave correctly, just that we don't end
-    // up with any errors
-    testApplabConsoleOutput({
-      testName: 'Turtle',
-      source: `
-        moveForward(25);
-        moveBackward(25);
-        move(25, 25);
-        moveTo(0, 0);
-        dot(5);
-        turnRight(90);
-        turnLeft(90);
-        turnTo(0);
-        arcRight(90, 25);
-        arcLeft(90, 25);
-        getX();
-        getY();
-        getDirection();
-        penUp();
-        penDown();
-        penWidth(3);
-        penColor("red");
-        penRGB(120, 180, 200);
-        show();
-        hide();
-        speed(50);
```

Comment:
ahh, you're right -- i thought this was covered in [ec_math.js](https://github.com/code-dot-org/code-dot-org/blob/c1a902a3cd46276d7e9a77fc98ab0674af794d81/apps/test/integration/levelSolutions/applab1/ec_math.js#L14) because the comment says, "These exercise all of the blocks in Turtle category," but it looks like the comment is just wrong, so i'll update it